tests/ExcelProvider.Tests/ExcelProvider.Tests.fs

```diff
@@ -57,6 +57,7 @@ type MultiLine = ExcelFile<"MultilineHeader.xlsx">
 type MultipleSheetsFirst = ExcelFile<"MultipleSheets.xlsx", "A">
 type MultipleSheetsSecond = ExcelFile<"MultipleSheets.xlsx", "B">
 type MultipleSheetsSecondRange = ExcelFile<"MultipleSheets.xlsx", "B", "A2">
+type MixedDataTypes = ExcelFile<"MixedDataTypes.xlsx">
```

Comment:
The file in question looks like this:
![image](https://github.com/user-attachments/assets/336f05dc-d393-488e-964a-b2324dcedb30)
Annotations applying to the Title column only.